src/test/java/util/xcmailr/util/SendRequest.java

```diff
@@ -48,10 +47,11 @@ public static void login(String email, String password)
 
```

Comment:
Don't declare variables in a line and then set a value right after. Just do both in a singe line, e.g.
```
HttpResponse  = httpClient.execute(postRequest);
```